docs/install_guide.adoc

```diff
@@ -0,0 +1,140 @@
+= Installation Guide
```

Comment:
It looks great. We need to add[ main.adoc file](https://raw.githubusercontent.com/openstack-k8s-operators/data-plane-adoption/refs/heads/main/docs_dev/main.adoc)
```
= Watcher Operator documentation
:toc: left
:toclevels: 3
:icons: font
:compat-mode:
:doctype: book
:context: osp

include::install_guide.adoc[leveloffset=+1]
```
We also need to fix this part https://github.com/openstack-k8s-operators/watcher-operator/commit/214e8bd24543f7cb87f78105d148e9c012835a02#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R460 Like this.
```
bundle exec kramdoc README.md -o docs/readme.adoc
```
And also update the above main.adoc changes to include above readme
```
include::readme.adoc[leveloffset=+1]
```
It will generate all the docs.